corehq/apps/es/es_query.py

```diff
@@ -194,6 +194,11 @@ def add_filter(*args, **kwargs):
                 return add_filter
         raise AttributeError("There is no builtin filter named %s" % attr)
 
+    def __getitem__(self, sliced):
+        start = sliced.start or 0
+        size = sliced.stop - start
+        return self.start(start).size(size).run()
```

Comment:
neat. should this also support integer indexing? might be confusing that it only takes slices.